pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
 
 <%= toc %>
 
-A *command* step runs a command on an agent. The following is the most simple *command* step:
+
+A *command* step runs a command on an agent. The command can be the name of a script, or <cli things>
```

Comment:
The <cli things> bit was waiting for more info - I wasn't really sure what to call it. You can put commands in there that you would normally run on the command line, right? So like, you could cd somewhere and then do some things? I wanted to mention that you could do that there and that it doesn't have to just be the name of a script.